docs/napari-plugin-guide.md

```diff
@@ -10,120 +11,163 @@ Launch `napari` with `recOrder`
+## Acquisition / Reconstruction Tab
+This acquisition tab is designed to acquire and reconstruct single volumes of both phase and birefringence measurements to allow the user to test their calibration and background. We recommend this tab for quick testing and the Micromanager MDA acquisition for high-throughput data collection.
 
-### Example of Successful Calibration
-Below is an example of a successful calibration with a reasonable extinction value:
+### Acquire Buttons
+![](./images/acquire_buttons.png)
 
-![](https://github.com/mehta-lab/recOrder/blob/main/docs/images/calib_finished.png)
+The **Retardance + Orientation**, **Phase From BF**, and **Retardance + Orientation + Phase** buttons set off MicroManager acquisitions that use the upcoming acquisition settings. After the acquisition is complete, these routines will set off `recOrder` reconstructions that estimate the named parameters. 
 
-## Acquisition / Reconstruction Tab
-This acquisition module is designed to take single image volumes for both phase and birefringence measurements and allows the user to test the outcome of their calibration with small snapshots or volumes. We recommend this acquisition mode for quick testing and the Micromanager MDA acquisition for high-throughput data collection.
-
-### Save Path
-User specifies the directory in which the images will be saved.
+The **STOP** button will end the acquisition as soon as possible, though MicroManager acquisitions cannot always be interrupted. 
 
 ### Acquisition Settings
-*Z Start, Z End, Z Step* specify the relative z-parameters to use for acquiring an image volume. Values are in the default units of the stage, typically in microns. The center slice will be the current position of the z-stage
+![](./images/acquisition_settings.png)
+
+The **Acquisition Mode** sets the target dimensions for the reconstruction. Perhaps surprisingly, all 2D reconstructions require 3D data except for **Retardance + Orientation** in **2D Acquisition Mode**. The following table summarizes the data that will be acquired when an acquisition button is pressed in **2D** and **3D** acquisition modes:
 
-![](https://github.com/mehta-lab/recOrder/blob/main/docs/images/acquisition.png)
+| **Acquisition** \ Acquisition Mode | 2D mode | 3D mode |  
+| :--- | :--- | :--- |
+| **Retardance + Orientation** | CYX data | CYX data | 
+| **Phase From BF** | ZYX data | ZYX data | 
+| **Retardance + Orientation + Phase** | CZYX data | CZYX data | 
 
-Ex. for a 20 um thick cell, the user would first focus in the middle of the cell and then set the following parameters: 
+Unless a **Retardance + Orientation** reconstruction in **2D Acquisition Mode** is requested, `recOrder` uses MicroManager's z-stage to acquire 3D data. **Z Start**, **Z End**, and **Z Step** are stage settings for acquiring an image volume, relative to the current position of the stage. Values are in the stage's default units, typically in microns.
```

Comment:
```suggestion
Unless a **Retardance + Orientation** reconstruction in **2D Acquisition Mode** is requested, `recOrder` uses MicroManager's z-stage to acquire 3D data. **Z Start**, **Z End**, and **Z Step** are stage settings for acquiring an image volume, relative to the current position of the stage. Values are in the stage's default units, typically in micrometers.
```

Reply:
Thanks, changed all "microns" to "micrometers".